tests/testcases/example-app/cmd/coin.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
```

Comment:
Maybe you could either remove this, or use your name

Reply:
that's just a test application, we can leave it like so